broadcast/broadcast-client/client_builder_mocks.go

```diff
@@ -0,0 +1,39 @@
+package broadcast_client
```

Comment:
I would rather use a separated package like `mock_broadcast_client` or `broadcast_client_test` or something that is showing that it's for testing purposes.

And also IMHO we should provide a separated builder for tests.

Reply:
Also it would be useful to add some instructions in the go docs of this new builder.

Reply:
Done